### ISSUE TITLE
[editor] Add OperationsEditor

### ISSUE THREAD

**Issue by dotansimha** [the-guild-org/the-guild-components#49]
We need `OperationsEditor` that has "Schema Awareness" and does validation based on the GraphQL schema. 

Use-cases:
- Envelop Playground
- Codegen live-demo

We should be able to connect the `OperationsEditor` to `SchemaEditor`, something like: 

```
const [schema, setSchema] = useState<GraphQLSchema>(null);

return (
   <div>
     <SchemaEditor value="" onSchemaChange={schema => setSchema(schema)} />
     <OperationsEditor value="" schema={schema} />
  </div>
);
```

**Comment by n1ru4l** [comment]
Gonna link this here https://github.com/dotansimha/envelop/pull/1052

**Comment by B2o5T** [comment]
@n1ru4l `<ExecutableDocumentEditor />` available in `@theguild/editor@0.5.0`
